AccountTransferApi/build.gradle

```diff
@@ -2,10 +2,11 @@
 
 buildscript {
     repositories {
-        jcenter()
+        google()
+        mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.3'
+        classpath 'com.android.tools.build:gradle:4.2.0'
```

Comment:
I think 4.2.1 the latest at this moment.

Reply:
done